projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/dynamic-radio-list/write-dynamic-radio-list-field.component.ts

```diff
@@ -19,7 +19,7 @@ export class WriteDynamicRadioListFieldComponent extends AbstractFieldWriteCompo
     }
 
     if (!this.caseField.value && this.caseField.formatted_value && this.caseField.formatted_value.value) {
```

Comment:
The if statement can be further simplified - if (!this.caseField?.value && this.caseField?.formatted_value?.value) { }.